spec/consensus/proposer-based-timestamp/pbts-sysmodel.md

```diff
@@ -71,6 +71,27 @@ proposal message broadcast by correct processes: it is a *worst-case* parameter.
 As message delays depends on the message size, the above requirement implicitly
 indicates that the size of proposal messages is either fixed or upper bounded.
 
+#### **[PBTS-MSG-DELAY-ADAPTIVE.0]**
+
+A practical relaxation of [PBTS-MSG-DELAY.0] is to consider adaptive end-to-end
+delays, namely, that `MSGDELAY` increases each time consensus requires a new round.
+In this way, after a number of rounds the observed end-to-delay should match
+the adopted `MSGDELAY`. This is typical for latency parameters in partial
+synchronous models.
+
+The adaptive system parameter `MSGDELAY(r)` is defined as follows.
+Lets `p` and `q` be any correct processes:
+
+- If `p` sends a proposal message `m` from round `r` at real time `t` and `q` receives `m` at
+  real time `t'`, then `t <= t' <= t + MSGDELAY(r)`.
+
+The adaptiveness is represented by the assumption that the value of the
+parameter increases over rounds, i.e., `MSGDELAY(r+1) > MSGDELAY(r)`.
+The initial value `MSGDELAY(0)` is equal to `MSGDELAY` as in [PBTS-MSG-DELAY.0].
+
+For the sake of correctness and formal verification, having a fixed and
+well-chosen `MSGDELAY` is enough.
```

Comment:
I am not sure what you want to say here. I guess I would say that if `MSGDELAY` is chosen sufficiently large, then the fact that it increments in later rounds (1) in practice will never be experienced, and (2) also has no theoretical implications. The increment is here to handle potential misconfigurations.